Products/ZenEvents/zentrap.py

```diff
@@ -545,17 +545,21 @@ def _value_from_dateandtime(self, value):
                 mon, day, hour, mins, secs, dsecs, utc_dir, utc_hours, utc_mins)
 
     def _convert_value(self, value):
-        if not isinstance(value, basestring):
-            return value
-        try:
-            value.decode('utf8')
-            return value
-        except UnicodeDecodeError:
-            # Try converting to a date
-            decoded = self._value_from_dateandtime(value)
-            if not decoded:
-                decoded = 'BASE64:' + base64.b64encode(value)
-            return decoded
+        # Check if values is OID (SNMP object id)
+        if isinstance(value, tuple) and len(value) > 2:
```

Comment:
> It's only an OID if it has at least 3 elements in the tuple?
> Does it have to be a tuple? Could it be any kind of sequence?
Only OIDs are stored in tuples.

> What's the scenario for 1 and 2 element tuples?
Will not happen. OID minimum length is 3 (start with 1.3.). Can we call this 'right way'? This code will be working with `if isinstance(value, tuple)` too.